src/calculateRentalCost.js

```diff
@@ -29,6 +29,13 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 3 && days < 7) {
+    return (40 * days) - 20;
+  } else if (days >= 7) {
+    return (40 * days) - 50;
+  } else {
```

Comment:
you should read the checklist about else and variables.
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md